metadata-ingestion/src/datahub/ingestion/source/aws/dynamodb.py

```diff
@@ -174,8 +173,7 @@ def populate_attribute_definitions(self, table) -> Dict[str, str]:
                 attribute_definitions[attr] = new_value
         else:
             table_name = table["TableName"]
-            logger.warn(f"Table {table_name} streaming is NOT enabled.")
-            self.report.report_table_streaming_disabled(table_name)
```

Comment:
Can we also clean up this function in line 79 as that is no longer used anywhere.